### ISSUE TITLE
ACC-264.1 (Level A)_ List _ Associated label not announced.

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3243]
v1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	List
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1 (Level A)_ List _ Associated label not announced.

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Go to KubeconfigURL:https://kyma-env-broker.cp.stage.kyma.cloud.sap/kubeconfig/DFEFE4E0-E202-4455-8A61-37511B4D7553
•	Click on  the link https://dashboard.stage.kyma.cloud.sap/clusters and select Connect.
•	Upload the kubeconfing file downloaded in the first step. You can either drag and drop it or select the upload icon to choose it from the device.
•	Next Step
•	Connect to cluster, A new screen get opens
•	Open ‘Profile account select ‘Preference’ button 
•	

3. Issue Type: Manual
Observed Behavior:
On accessing the Listbox associated label such as ‘Select default view for resource edit’ is not read out Within the dialog ‘Preference’ .

Screenshot:
 
Expected Behavior:
Only relevant information about the UI should be communicated to the user. Additional information should be avoided.

[ACC-264.1 (Level A)_ List _ Associated label not announced.docx](https://github.com/user-attachments/files/16687591/ACC-264.1.Level.A._.List._.Associated.label.not.announced.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16687592/Issue.processing-09.2021.2.docx)
